### ISSUE TITLE
jasmine-auto-spies@>=6.9.2 breaks the test suite in Internet Explorer

### ISSUE THREAD

**Issue by ThomasOrtiz** [hirezio/auto-spies#45]
**Describe the bug**
Starting in version `6.9.2` of `jasmine-auto-spies` I receive an error in Internet Explorer _(see screenshot 1)_ that causes the test suite to break. 

If I downgrade the library to `6.9.1` the tests will pass as expected.

Version:
- `<=6.9.1` works fine in IE and other browsers
- `>=6.9.2` works fine in non-IE browsers.

**To Reproduce**
1. Upgrade jasmine-auto-spies to `>=6.9.2`.
2. Use jasmine-auto-spies in a test. The readme's `addTwoNumbers` example _(snippet below)_ will exhibits the same behavior.
3. Run your tests in Internet Explorer.
4. Open the console and see the console error.

Example test:
```TS
describe('Test Spec', () => {
  function addTwoNumbers(a: number, b: number): number {
    return a + b;
  }
  const functionSpy = createFunctionSpy<typeof addTwoNumbers>('addTwoNumbers');

  it('should be defined', () => {
    expect(functionSpy).toBeDefined();
  });
});

```

**Expected behavior**
I expected the tests to work across all browsers.

**Screenshots**
**Screenshot 1**
![image](https://user-images.githubusercontent.com/10877317/111104437-18ba1880-8527-11eb-942a-9e48d67e0126.png)

**Desktop:**

- OS: Windows 10
- Browser: Internet Explorer
- Version: 11

**Additional context**
[6.9.1 vs 6.9.2 changeset](https://github.com/hirezio/auto-spies/compare/jasmine-auto-spies%406.9.1...jasmine-auto-spies%406.9.2)
  - I noticed `code-stringify` was added in the `6.9.2` release which is also the same portion that the bundle blows up at in _Screenshot 1_.
  - ![image](https://user-images.githubusercontent.com/10877317/111103760-8ebd8000-8525-11eb-8df4-76859834f65c.png)

**Comment by shairez** [comment]
Thanks for the report @ThomasOrtiz !

I believe this bug happens when you're using `calledWith` or `mustBeCalledWith` right?

I tried to connect this library's CI to browserstack so we could test it under explorer 11 but I ran into some issues and then another deadline pressed me and I had to pause the work on that. (just now coming back to deal with repo issues)

I'm currently still swamped with work, so I would appreciate any help I could get

Basically I had to change to `code-stringify` to fix bug #39 

So if you (or anyone else) can help find a different library that can "stringify" the parameters code, and issue a PR that would move this fix forward.

If not, you can avoid using `calledWith` or `mustBeCalledWith` for non evergreen browsers and that should be a temporary fix until the PR.

Thanks!

**Comment by shairez** [comment]
Please let me know if this bug persists in the latest version
If so, please re-open this issue
Thanks
